### DISCUSSION TITLE
Github actions for solution that include themes

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#14207] [General]
I'm attempting to use github actions to build and publish my code to an azure app.  I get an error using the yml supplied by visual studio because the project file for the theme is not with the source for the cms.  Is there a way that I could include the theme as a dll or should it I be looking to make a change in the yml?  I'm new to using github actions, so any advice would be useful.

**Comment by ns8482e** [comment]
Are you working and building on Orchard Core Source code? If that's the case you can add your theme project's reference to Orchard's Cms Web project.

IMO The better way to build your app is not use orchard source directly- instead create your own solution and project as any other .net projects and add orchard core as nugget reference

**Comment by ghost** [comment]
I run into an error when the action is attempting to build because it can't seem to find the skin solution.

C:\Program Files\dotnet\sdk\7.0.400\NuGet.targets(400,5): error MSB3202: The project file "<path>Theme.csproj" was not found. [<path>.Web.sln]
